packages/avida_test_utils/src/sdjwt/fixtures.rs

```diff
@@ -3,13 +3,43 @@ use sd_jwt_rs::SDJWTIssuer;
 use serde_json::Value;
 use std::{fs, path::PathBuf};
 
+use cosmwasm_std::Binary;
+
+use sylvia::multitest::{App, Proxy};
+
+use avida_common::types::{
+    InputRoutesRequirements, RouteVerificationRequirements, VerificationSource,
+};
+use avida_sdjwt_verifier::{
+    contract::sv::mt::CodeId,
+    contract::*,
+    types::{Criterion, InitRegistration, MathsOperator, PresentationReq},
+};
+use josekit::{self};
+
+use cw_multi_test::App as MtApp;
+
+/// Test constants
+pub const OWNER_ADDR: &str = "addr0001";
+pub const FIRST_CALLER_APP_ADDR: &str = "addr0002";
+pub const SECOND_CALLER_APP_ADDR: &str = "addr0003";
+
+pub const VERIFIER_CONTRACT_LABEL: &str = "Verifier Contract";
+
+pub const FX_ROUTE_ID: u64 = 1;
```

Comment:
Initially I had this `FX` prefix just to identify as fixtures, but it is redundant as it is in the `fixtures` file. 

I think we should rename to FIRST